src/main/java/randoop/operation/MethodCall.java

```diff
@@ -254,8 +254,6 @@ public static TypedClassOperation parse(String signature) throws OperationParseE
       throw new OperationParseException(msg);
     }
 
-    System.out.println("Looking for: " + opname + " args: " + arguments);
```

Comment:
I would prefer to add lines as needed for particular debugging scenarios, and remove them when done. Otherwise, there would either be a lot of these messages to dredge through, or very complex mechanisms for enabling relevant groups of messages.